packages/firestore/src/local/indexeddb_schema_converter.ts

```diff
@@ -240,9 +253,9 @@ export class SchemaConverter implements SimpleDbSchemaConverter {
     }
 
     if (fromVersion < 14 && toVersion >= 14) {
-      p = p.next(() => {
-        createFieldIndex(db);
-      });
+      p = p
+        .next(() => createFieldIndex(db))
+        .next(() => this.runOverlayMigration(db, simpleDbTransaction));
```

Comment:
Your overlay migration should run as version 14 and then indexing should be version 15. That way, we can enable overlays soon.